templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -458,6 +452,12 @@ Parameters:
     AllowedValues: [General Purpose (SSD), Provisioned IOPS]
     ConstraintDescription: Must be 'General Purpose (SSD)' or 'Provisioned IOPS'.
     Type: String
+  InternetFacingLoadBalancer:
+    Default: true
+    AllowedValues: [true, false]
+    ConstraintDescription: Must be 'true' or 'false'.
+    Description: Controls if the load balancer is deployed as internet-facing (visible to all on internet) or internal (private to VPC).
```

Comment:
I'd suggest:

> Controls whether the load balancer should be visible to the internet (true) or only within the Atlassian Standard Infrastructure/VPC (false).

It's explicit as to what each boolean does. Also, not sure if we want to specify the ASI or simply say VPC?

Reply:
Lets say VPC because that's more correct

Reply:
Technically the customer could deploy anything else into the same VPC as ASI and that would be able to see load balancer